gravity-forms/gw-progress-meter.php

```diff
@@ -51,7 +51,9 @@ public function do_meter_shortcode( $output, $atts ) {
 			$count = $atts['start'];
 		}
 
-		$percent_complete = $count <= 0 ? 0 : round( ( $count / $atts['goal'] ) * 100 );
+		$goal = $this->get_goal( $count, $atts['goal'] );
+
+		$percent_complete = $count <= 0 ? 0 : round( ( $count / $goal ) * 100 );
```

Comment:
Does `$goal` need to be converted to an int/float before performing math?

Reply:
I can dig it!

Reply:
Dank!